mep3_simulation/webots_data/worlds/eurobot_2022.wbt

```diff
@@ -583,6 +583,21 @@ DEF ROBOT_OPPONENT_SMALL OpponentBox {
   rotation 0 0 1 0
   arucoNumber 2
 }
+Robot {
+  translation -0.1 1.00976 1.05
+  rotation 0.4472158700299192 0.4472128700307911 -0.7745957748866213 1.82349
+  children [
+    JetBotRaspberryPiCamera {
```

Comment:
Ja bih da izbacimo ovaj PROTO, dovoljeno je samo Camera uredjaj da ubacimo. Simulacija bi trebalo da bude sto "laksa"

Reply:
Da li si siguran u ovo? Pokušao sam sad da ubacim samo `JetBotRasberryCamera.proto` u `eurobot_2023.wbt`, nije mogla da se ubaci kamera, samo su se `Transform`-i ubacili.

`WARNING: JetBotRaspberryPiCamera (PROTO): Skipped node: Cannot insert Camera node in 'children' field of Transform node that doesn't have a Robot ancestor.`